skaha/VERSION

```diff
@@ -1,4 +1,4 @@
 ## deployable containers have a semantic and build tag
 # semantic version tag: major.minor
 # build version tag: timestamp
-TAGS="0.10.0 $(date -u +"%Y%m%dT%H%M%S")"
+TAGS="0.10.1 $(date -u +"%Y%m%dT%H%M%S")"
```

Comment:
Did you pull in `master`?  The current version is 0.11.0.

Reply:
Yep, did another pull from master and updated to version 0.11.1.